From 19e69ad23240e19ed70202692ebd4099da7f7e8f Mon Sep 17 00:00:00 2001
From: Roger Monk <r-monk@ti.com>
Date: Tue, 14 Aug 2012 18:06:13 +0100
Subject: [PATCH 087/102] mux33xx: Fix MUXENTRYs for MCASP0_ACLKX/FSX to add
 ehrpwm0 and pru pins

   * Add Mux Entries for ehrpwm0 to MCASP0_CLKX/FSX pins
   * Also add modes 5/6 to the same pins for completeness (pru pins)

Signed-off-by: Roger Monk <r-monk@ti.com>
---
 arch/arm/mach-omap2/mux33xx.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-omap2/mux33xx.c b/arch/arm/mach-omap2/mux33xx.c
index 852f7f0..c0d14b3 100644
--- a/arch/arm/mach-omap2/mux33xx.c
+++ b/arch/arm/mach-omap2/mux33xx.c
@@ -335,11 +335,11 @@ static struct omap_mux __initdata am33xx_muxmodes[] = {
 		"i2c0_scl", NULL, NULL, NULL,
 		NULL, NULL, NULL, "gpio3_6"),
 	_AM33XX_MUXENTRY(MCASP0_ACLKX, 0,
-		"mcasp0_aclkx", NULL, NULL, "spi1_sclk",
-		"mmc0_sdcd", NULL, NULL, "gpio3_14"),
+		"mcasp0_aclkx", "ehrpwm0A", NULL, "spi1_sclk",
+		"mmc0_sdcd", "pr1_pru0_pru_r30_0", "pr1_pru0_pru_r31_0", "gpio3_14"),
 	_AM33XX_MUXENTRY(MCASP0_FSX, 0,
-		"mcasp0_fsx", NULL, NULL, "spi1_d0",
-		"mmc1_sdcd", NULL, NULL, "gpio3_15"),
+		"mcasp0_fsx", "ehrpwm0B", NULL, "spi1_d0",
+		"mmc1_sdcd", "pr1_pru0_pru_r30_1", "pr1_pru0_pru_r31_1", "gpio3_15"),
 	_AM33XX_MUXENTRY(MCASP0_AXR0, 0,
 		"mcasp0_axr0", NULL, NULL, "spi1_d1",
 		"mmc2_sdcd", NULL, NULL, "gpio3_16"),
-- 
1.8.1.4

